Source/MVC6/Boilerplate.AspNetCore.TagHelpers/Twitter/TwitterCard.cs

```diff
@@ -58,31 +97,27 @@ public override string ToString()
         }
 
         /// <summary>
-        /// Appends a HTML-encoded string representing this instance to the <paramref name="stringBuilder"/> containing
-        /// the Twitter card meta tags.
+        /// Appends a HTML-encoded string representing this instance to the <paramref
+        /// name="stringBuilder"/> containing the Twitter card meta tags.
         /// </summary>
         /// <param name="stringBuilder">The string builder.</param>
         public virtual void ToString(StringBuilder stringBuilder)
         {
             this.Validate();
 
-            if (this.Type != TwitterCardType.Summary)
+            stringBuilder.AppendMetaNameContent("twitter:card", this.Type.ToTwitterString());
```

Comment:
`twitter:card` was only required if the card type was not a summary. Has that changed?